benchmark/docker-compose.yml

```diff
@@ -66,7 +66,6 @@ services:
     - --rpc-ws-enabled
     - --rpc-ws-apis=admin,eth,miner,web3,net,priv,eea
     - --rpc-ws-host=0.0.0.0
-    - --host-whitelist=*
```

Comment:
instead of deleting this line, can you change to host-allowlist

Reply:
Alright, I changed it to host-allowlist